core/src/main/java/io/questdb/cairo/wal/WalWriter.java

```diff
@@ -1308,6 +1339,7 @@ private void setVarColumnFixedFileNull(int columnType, int columnIndex, long row
             } finally {
                 ff.munmap(addressFixed, fixedSizeColSize, MEM_TAG);
             }
+            ff.fsync(fixedSizeColumn.getFd());
```

Comment:
We should consider commit mode from config here and in similar calls below.

Reply:
fixed here https://github.com/questdb/questdb/pull/3304/commits/63cf4210b42daeb8118233d89abbe70b18fe41a4

also replaced `fsync` with `msync`